cmd/cluster/resizecontrolplanenode.go

```diff
@@ -253,6 +253,8 @@ func (o *resizeControlPlaneNodeOptions) forceDrainNode(nodeID string, reason str
 	// As RunElevate is unsetting KUBECONFIG, we need to redefined it if it was defined
```

Comment:
Could `os.Unsetenv("KUBECONFIG")` be placed above all the if statements?

Reply:
Sorry, I do not see the point of unsetting the variable if we set it to a value?
And I guess it is more readable, as:
 if variable was set,
 then we set it to old value,
 else we unset it to retrieve the original configuration...

Reply:
Instead of copying the same line of code three places, you can use it once and get rid of unnecessary elses.